pkgs/tools/security/krunner-pass/default.nix

```diff
@@ -0,0 +1,36 @@
+{ mkDerivation, stdenv,
+  fetchFromGitHub,
+  cmake, extra-cmake-modules, gnumake,
+
+  pass, pass-otp ? null, krunner,
```

Comment:
Why `pass-otp` is optional? It seems to me this is not necessary

Reply:
Judging by it's [readme](https://github.com/akermu/krunner-pass/blob/master/README.md) it can optionally be used with `pass-otp` hence the optional dependency.

Reply:
ok, fine.